src/pages/_assets/welcome-content.tsx

```diff
@@ -0,0 +1,263 @@
+import { CardItem } from "../../components/WelcomeCard/welcome-card"
+
+// Section 1: Identity Management
+export const identityManagement: {
+  id: string
+  title: string
+  description: string
+  cards: CardItem[]
+} = {
+  id: "identity-management-section",
+  title: "Identity Management",
+  description: "What your users need for authentication and user management",
+  cards: [
+    {
+      title: "Implement User Login",
```

Comment:
This should be in sentence casing: https://github.com/ory/docs?tab=readme-ov-file#headings